docs/configuration.rst

```diff
@@ -260,18 +75,17 @@ Destroys a dataset in the given repository with a given name.
 
 .. code-block:: bash
 
-    $ ga4gh_repo remove-dataset path/to/datarepo aDataset
+    $ ga4gh_repo remove-dataset path/to/repo.db aDataset
```

Comment:
I think the time for these comments was in the #1158 PR @david4096. This was never meant to be a full and final update of the docs. The stated intention was to make them less inaccurate. We definitely want to do a full run through of the docs once this has been merged, but I don't think it makes sense to block the whole PR on it.